lib/imports.js

```diff
@@ -46,10 +48,10 @@ function updateImports(imports, mod, identifier, type) {
       return;
     case 'method':
       if (!imports.Reflect) {
-        imports.Reflect = {namedImports: new Set(['call as Reflect_call'])};
+        imports.Reflect = {namedImports: new Set(['apply as Reflect_apply'])};
       }
-
-      imports.Reflect.namedImports.add('call as Reflect_call');
+      imports.Reflect.namedImports.add('apply as Reflect_apply');
+    case 'static':
```

Comment:
Don't use switch fall-through in this way. Put the `break` before the `case`, and then duplicate the `imports[mod].namedImports.add(importText);` line. (In this case I think clarity outweighs avoiding duplication.)

Also it would be good to switch from `return` to `break` in general.

Reply:
Yeah I had this trick in when it helped prevent code duplication but it has since outstayed its welcome, fixing now